src/pwm.rs

```diff
@@ -1233,16 +1276,16 @@ macro_rules! tim_hal {
                 $(
                     /// Set the deadtime for complementary PWM channels of this timer
                     #[must_use]
-                    pub fn with_deadtime<T: Into<NanoSeconds>>(mut self, deadtime: T) -> Self {
+                    pub fn with_deadtime(mut self, deadtime: NanoSeconds) -> Self {
```

Comment:
Accepting `NanoSeconds` instead of `Into<Nanoseconds>` is less flexible and a breaking change. Could it be left as before?

Reply:
reverted